zgw/object-management/src/main/kotlin/com/ritense/objectmanagement/service/ObjectManagementFacade.kt

```diff
@@ -0,0 +1,203 @@
+package com.ritense.objectmanagement.service
+
+import com.fasterxml.jackson.databind.JsonNode
+import com.ritense.objectenapi.ObjectenApiPlugin
+import com.ritense.objectenapi.client.ObjectRecord
+import com.ritense.objectenapi.client.ObjectRequest
+import com.ritense.objectenapi.client.ObjectWrapper
+import com.ritense.objectenapi.client.ObjectsList
+import com.ritense.objectmanagement.domain.ObjectManagement
+import com.ritense.objectmanagement.repository.ObjectManagementRepository
+import com.ritense.objecttypenapi.ObjecttypenApiPlugin
+import com.ritense.plugin.service.PluginService
+import mu.KotlinLogging
+import org.springframework.data.domain.PageRequest
+import org.springframework.stereotype.Service
+import org.springframework.web.reactive.function.client.WebClientResponseException
+import java.net.URI
+import java.time.LocalDate
+import java.util.UUID
+
+@Service
+class ObjectManagementFacade(
+    private val objectManagementRepository: ObjectManagementRepository,
+    private val pluginService: PluginService
+) {
+    val logger = KotlinLogging.logger {}
```

Comment:
Please make the logger private and move it to a `companion object`

Reply:
This has been done.